src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Microsoft.Extensions.Configuration.Binder.SourceGeneration.Tests.csproj

```diff
@@ -49,14 +49,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="Baselines\net462\*.generated.txt;
-                      Baselines\net462\ConfigurationBinder\*.generated.txt;
-                      Baselines\net462\OptionsBuilder\*.generated.txt;
-                      Baselines\net462\ServiceCollection\*.generated.txt;
-                      Baselines\netcoreapp\*.generated.txt;
-                      Baselines\netcoreapp\ConfigurationBinder\*.generated.txt;
-                      Baselines\netcoreapp\OptionsBuilder\*.generated.txt;
-                      Baselines\netcoreapp\ServiceCollection\*.generated.txt;">
+    <Content Include="Baselines\*\*\*.generated.txt;
```

Comment:
this doesn't look right.

Reply:
seems right to me. There qre are 2 patterns :
* Baselines\netcoreapp\OptionsBuilder\*.generated.txt => which would translate to Baselines\*\*\*.generated.txt
* Baselines\netcoreapp\Primitives.txt => which would translate to Baselines\*.generated.txt

Reply:
If we can copy the whole Baseline as I pointed out in https://github.com/dotnet/runtime/pull/89952#discussion_r1353284799 that will be better I guess.